src/dojos/cd-find-dojo.vue

```diff
@@ -138,6 +138,13 @@
         this.coordinates.longitude = this.long;
         this.getDojosByLatLong();
       }
+      if (this.$route.query.q) {
+        this.searchCriteria = this.$route.query.q;
+        this.searchDojosByAddress();
+      }
+      if (this.$route.query.currentLocation) {
```

Comment:
Better to have `else if` here, because otherwise we could potentially end up loading two different sets of Dojos. I'd be inclined to take `q` over `currentLocation` as it's specific user input.